lib/Backend/arm64/LowerMD.cpp

```diff
@@ -1368,32 +1294,19 @@ LowererMD::LowerEntryInstr(IR::EntryInstr * entryInstr)
         this->m_func->m_unwindInfo.SetPrologStartLabel(prologStartLabel->m_id);
     }
 
-    int32 regPushedSize = 0;
+    // STP fp, lr, [sp, -preIndexedOffset]!
+    // STP r8 to r28 (in pairs), [sp, +ve offset]
+    // STP d16 to d29 (in pairs), [sp, +ve offset]
```

Comment:
Should only be saving r19 to r28. Please either update the comment or fix the logic.